test/manifests/fakerp/create.yaml

```diff
@@ -29,4 +29,5 @@ properties:
     vmSize: {{ if eq .RunningUnderTest "true" }}Standard_D2s_v3{{ else }}Standard_D4s_v3{{ end }}
     subnetCidr: 10.0.0.0/24
   monitorProfile:
+    enabled: true
```

Comment:
good.  I think the line below should be something like:
`    workspaceResourceId: {{ print "/subscriptions/" .SubscriptionID "/resourcegroups/defaultresourcegroup-" .Region "/providers/microsoft.operationalinsights/workspaces/DefaultWorkspace-" .Region | quote }}`